Assets/MRTK/Core/Providers/Hands/ArticulatedHandDefinition.cs

```diff
@@ -196,5 +226,71 @@ public void UpdateCurrentIndexPose(MixedRealityInteractionMapping interactionMap
                 }
             }
         }
+
+        // Used to track the input that was last raised
+        private Vector2 previousStickInput = Vector2.zero;
+        private bool previousReadyToTeleport = false;
+
+        private static readonly ProfilerMarker UpdateCurrentTeleportPosePerfMarker = new ProfilerMarker("[MRTK] ArticulatedHandDefinition.UpdateCurrentTeleportPose");
+
+        public void UpdateCurrentTeleportPose(MixedRealityInteractionMapping interactionMapping)
+        {
+            using (UpdateCurrentTeleportPosePerfMarker.Auto())
+            {
+                MixedRealityInputAction teleportAction = MixedRealityInputAction.None;
```

Comment:
Also looks like this isn't used anymore, now that the action is mapped

Reply:
Yeah, that was a consideration I had when renaming the variables back.

I think the teleportAction is still used by TeleportPointer to make sure the right input event is being raised

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/abb4aaa63859646de12a195f0566f8caa5999773/Assets/MRTK/SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs#L383